### PR TITLE
DOC-2363: Added new `A11ycheckStart`, `A11ycheckStop`, `A11ycheckIgnore`, `A11ycheckRepair` and `A11ycheckShowDetails` events..

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2363

Site: [Staging branch- Release notes](http://docs-feature-71-doc-2363tiny-10777.staging.tiny.cloud/docs/tinymce/latest/7.1-release-notes/#added-new-a11ycheckstart-a11ycheckstop-a11ycheckignore-a11ycheckrepair-and-a11ycheckshowdetails-events)
Site: [Staging branch- Events](http://docs-feature-71-doc-2363tiny-10777.staging.tiny.cloud/docs/tinymce/latest/a11ychecker/#events)

Changes:
* Added new A11ycheckStart, A11ycheckStop, A11ycheckIgnore, A11ycheckRepair and A11ycheckShowDetails events to `a11ychecker.adoc`, and added release notes entry for new added events.

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`
- [x] Files has been included where required `(if applicable)`

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/partials/events/a11ychecker-events.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[cols="1,1,2",options="header"]
+|===
+|Name |Data |Description
+|A11ycheckStart |N/A |Fired when the Accessibility Checker is `opened`
+|A11ycheckStop |N/A |Fired when the Accessibility Checker is `closed`
+|A11ycheckIgnore |N/A |Fired when the `Ignore` button is toggled
+|A11ycheckRepair |N/A |Fired when the `Repair` button is toggled
+|A11ycheckShowDetails |N/A |Fired when the `Click for more info` button is toggled
```
Comment (FarzadHayat) [inline modules/ROOT/partials/events/a11ychecker-events.adoc:8]:
```suggestion
|A11ycheckIgnore |N/A |Fired when the `Ignore` button is pressed
|A11ycheckRepair |N/A |Fired when the `Repair` button is pressed
|A11ycheckShowDetails |N/A |Fired when the `Click for more info` button is pressed
```

File: modules/ROOT/pages/7.1-release-notes.adoc [hunk 2]
```diff
@@ -57,11 +57,33 @@ For information on the **<Open source plugin name>** plugin, see xref:<plugincod
...
+The events include:
+
+[cols="1,2",options="header"]
+|===
+|Name |Description
+|A11ycheckStart |Fired when the Accessibility Checker is `opened`
+|A11ycheckStop |Fired when the Accessibility Checker is `closed`
+|A11ycheckIgnore |Fired when the `Ignore` button is toggled
+|A11ycheckRepair |Fired when the `Repair` button is toggled
+|A11ycheckShowDetails |Fired when the `Click for more info` button is toggled
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.1-release-notes.adoc:80]:
```suggestion
|A11ycheckIgnore |Fired when the `Ignore` button is pressed
|A11ycheckRepair |Fired when the `Repair` button is pressed
|A11ycheckShowDetails |Fired when the `Click for more info` button is pressed
```

File: modules/ROOT/pages/7.1-release-notes.adoc [hunk 3]
```diff
@@ -57,11 +57,33 @@ For information on the **<Open source plugin name>** plugin, see xref:<plugincod
...
-The {productname} 7.1 release includes an accompanying release of the **<Premium plugin name 1>** premium plugin.
+The {productname} 7.1 release includes an accompanying release of the **Accessibility Checker** premium plugin.
 
-**<Premium plugin name 1>** <Premium plugin name 1 version> includes the following <fixes, changes, improvements>.
+**Accessibility Checker** includes the following improvement.
+
+==== Added new `A11ycheckStart`, `A11ycheckStop`, `A11ycheckIgnore`, `A11ycheckRepair` and `A11ycheckShowDetails` events.
+// #TINY-10777
+
+{productname} introduces five new events within the **Accessibility Checker** premium plugin. These events capture usage statistics, offering valuable insights into the tool's internal usage for customers.
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.1-release-notes.adoc:69]:
```suggestion
{productname} {release-version} introduces five new events within the **Accessibility Checker** premium plugin. These events capture usage statistics, offering valuable insights into the tool's internal usage for customers.
```

File: modules/ROOT/pages/7.1-release-notes.adoc [hunk 4]
```diff
@@ -57,11 +57,33 @@ For information on the **<Open source plugin name>** plugin, see xref:<plugincod
 
 The following premium plugin updates were released alongside {productname} 7.1.
 
-=== <Premium plugin name 1> <Premium plugin name 1 version>
+=== Accessibility Checker
 
-The {productname} 7.1 release includes an accompanying release of the **<Premium plugin name 1>** premium plugin.
+The {productname} 7.1 release includes an accompanying release of the **Accessibility Checker** premium plugin.
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.1-release-notes.adoc:62]:
```suggestion
The {productname} {release-version} release includes an accompanying release of the **Accessibility Checker** premium plugin.
```

File: modules/ROOT/pages/7.1-release-notes.adoc [hunk 5]
```diff
@@ -57,11 +57,33 @@ For information on the **<Open source plugin name>** plugin, see xref:<plugincod
...
+|===
+|Name |Description
+|A11ycheckStart |Fired when the Accessibility Checker is `opened`
+|A11ycheckStop |Fired when the Accessibility Checker is `closed`
+|A11ycheckIgnore |Fired when the `Ignore` button is toggled
+|A11ycheckRepair |Fired when the `Repair` button is toggled
+|A11ycheckShowDetails |Fired when the `Click for more info` button is toggled
+|===
+
+[NOTE]
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.1-release-notes.adoc:83]: doesn't the integrator choose how to handle the event being fired?

**Review by FarzadHayat — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/7.1-release-notes.adoc (hunk 5 above)
Comment (kemister85) [reply modules/ROOT/pages/7.1-release-notes.adoc:83]: Good point @FarzadHayat 
@spocke - Should we consider adding details on adding callback function to there integration if they want to initialise these events?

File: modules/ROOT/pages/7.1-release-notes.adoc [hunk 6]
```diff
@@ -57,11 +57,33 @@ For information on the **<Open source plugin name>** plugin, see xref:<plugincod
...
+These events will only be visible within the developer console when each `event` is triggered.
+
+For information on the **Accessibility Checker** plugin, see: xref:a11ychecker.adoc[Accessibility Checker].
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.1-release-notes.adoc:86]:
```suggestion
For information on the **Accessibility Checker** plugin, see: xref:a11ychecker.adoc[Accessibility Checker].

For information on handling events in {productname}, see: xref:events.adoc[Events].
```

**Comment by FarzadHayat** [comment]
@kemister85 [95aabd1](https://github.com/tinymce/tinymce-docs/pull/3245/commits/95aabd15d4a6f98243318eeee037aa9e614de46d)

**Review by FarzadHayat — APPROVED** [review]

File: modules/ROOT/pages/7.1-release-notes.adoc (hunk 5 above)
Comment (spocke) [reply modules/ROOT/pages/7.1-release-notes.adoc:158]: Yes how to listen to events should be a general thing we document and what they do in the handle could be anything.

**Review by spocke — APPROVED** [review]

**Review by shanmen-tiny — APPROVED** [review]

**Pull request merged**